assets/src/activation-notice/app/components/step1.js

```diff
@@ -24,12 +24,12 @@ import { useCallback } from 'react';
  * WordPress dependencies
  */
 import { __, _x } from '@wordpress/i18n';
+import { createInterpolateElement as TranslateWithMarkup } from '@wordpress/element';
```

Comment:
Our `TranslateWithMarkup` and Gutenberg's `createInterpolateElement` are very different tools with different APIs.

To use `createInterpolateElement` properly, we need to change the code below to something like this:

```js
{
  createInterpolateElement(
    __('Read the<br/><a>Get Started story</a>', 'web-stories'),
   {
      a: (
        <Link
          href={demoStoryURL}
          onClick={onClick}
          target="_blank"
          rel="noreferrer"
        />
      ),
      br: <br/>
    }
  )
}
```